scripts/ci/generate_latest_docs.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Rebuild gh-pages branch, adapted from
+# https://github.com/google/dagger/blob/master/util/generate-latest-docs.sh
+
+HOME=/tmp/aaa
```

Comment:
Oops, that was just so I could test it on my local machine without dumping lots of crap directly into my home directory.  Might make sense just to do the whole thing in tmp regardless.